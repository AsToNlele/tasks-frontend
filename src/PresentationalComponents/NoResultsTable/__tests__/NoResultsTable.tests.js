//import React from 'react';
import { render } from '@testing-library/react';

import emptyRows from '../NoResultsTable';

jest.mock('../../../../api');

describe('NoResultsTable', () => {
  let type = 'items';

  it('should render correctly', async () => {
    const { asFragment } = render(emptyRows(type));

    expect(asFragment()).toMatchSnapshot();
  });
});
