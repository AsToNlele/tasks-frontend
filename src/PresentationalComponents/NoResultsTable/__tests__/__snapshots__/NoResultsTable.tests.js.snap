// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResultsTable should render correctly 1`] = `
<DocumentFragment>
  <div
    class="ins-c-table__empty"
  >
     
    <div
      class="pf-l-bullseye"
    >
      <div
        class="pf-c-empty-state"
      >
        <div
          class="pf-c-empty-state__content"
        >
          <h5
            class="pf-c-title pf-m-lg"
            data-ouia-component-id="OUIA-Generated-Title-1"
            data-ouia-component-type="PF4/Title"
            data-ouia-safe="true"
          >
            No matching undefined found
          </h5>
          <div
            class="pf-c-empty-state__body"
          >
            To continue, edit your filter settings and search again.
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
